### DISCUSSION TITLE
Added TMUX and autoupdate features

### DISCUSSION THREAD

**Discussion by neonexus** [neonexus/sails-react-bootstrap-webpack#130] [Announcements]
In case you missed it, I just released an update that includes a couple handy new shell scripts for remote servers.

* `self-update.sh` will run a `git pull`, an `npm install`, and compare the contents of the `assets` folder before the `pull` and after; if there was a change, or `.tmp/public` doesn't exist, it will run `npm run build`.

* `tmux.sh` will run the self updater, then if everything was successful, will lift the server in `PRODUCTION` mode inside of a TMUX session. If you don't know about TMUX, I highly recommend you check it out, it's a VERY handy tool.

I've also written a quick-n-dirty guide on how to utilize these new scripts on remote environments. Check out the [README](https://github.com/neonexus/sails-react-bootstrap-webpack#getting-setup-remotely) for more.

@MaheshkumarSundaram

**Comment by MaheshkumarSundaram** [comment]
@neonexus Thank you. I'll check them out.

**Reply by MaheshkumarSundaram** [reply]
@neonexus,

`self-update.sh` is very useful for me when I deploy. Quick and handy. Thank you very much. I adapted this to Windows.

`tmux` is indeed handy but unfortunately, my organization uses windows.
